scripts/scripts.js

```diff
@@ -13,6 +13,25 @@ import {
   loadCSS,
 } from './aem.js';
 
+const experimentationConfig = {
+  prodHost: 'www.my-site.com',
+  audiences: {
+    mobile: () => window.innerWidth < 600,
+    desktop: () => window.innerWidth >= 600,
+    // define your custom audiences here as needed
+  },
+};
+
+window.hlx.plugins.add('experimentation', { // use window.hlx instead of your project has this
```

Comment:
I would not use the plugin system in our boilerplate, but rather use the manual instrumentation instead